certificate-renewal/Certificate Renewal JSON contract.md

```diff
@@ -162,8 +187,10 @@
 }
 ```
  - userCharList[1]: 10-digit account number, 'S' or 'L' and 4-digit share/Loan ID
+	 - Represents the certificate or club share being updated by the member
+ - userCharList[2]: 10-digit account number, 'S' or 'L' and 4-digit share/Loan ID
 	 - Represents the source or target share/loan of the transfer
- - userCharList[2]: User selected option
+ - userCharList[3]: User selected option
 	 - The transfer in/out amount
```

Comment:
@tkainz the amount being transferred out is never calculated in the UI

Reply:
true....although I suppose the member 'could' do a partial transfer out of funds and then let the cert roll over... but that's not an option we're offering.